connector/authproxy/authproxy.go

```diff
@@ -47,11 +48,21 @@ func (m *callback) HandleCallback(s connector.Scopes, r *http.Request) (connecto
 	if remoteUser == "" {
 		return connector.Identity{}, fmt.Errorf("required HTTP header X-Remote-User is not set")
 	}
-	// TODO: add support for X-Remote-Group, see
-	// https://kubernetes.io/docs/admin/authentication/#authenticating-proxy
+	remoteGroups, ok := r.Header["X-Remote-Group"]
+	if !ok {
+		remoteGroups = []string{}
```

Comment:
I don't think this is necessary: `nil` slices have a perfectly valid `empty slice` semantic in Go.

It probably doesn't hurt, but it doesn't help either I think.

Reply:
The json presentation of the nil will be null, which may not the for the client, therefore I enforced empty list.